src/components/Button/styles.css

```diff
@@ -8,28 +8,27 @@
 	line-height: 1;
 }
 
-.storybook-button--primary {
-	color: white;
-	background-color: #1ea7fd;
+.primaryButton {
+	background-color: #FFBD59;
```

Comment:
can we add this color as a CSS variable and use it instead?

Reply:
OK, would make the required changes